test/test_helper.rb

```diff
@@ -40,6 +40,11 @@
 module Warning
   class << self
     def warn(message)
+      # gems/selenium-webdriver-4.7.1/lib/selenium/webdriver/remote/bridge.rb:633:
```

Comment:
the warning is not under our control and it's not causing test failures so I decided to ignore it until we or someone else addresses it upstream

Reply:
It might be nice to raise an error if it doesn't happen anymore so we know when we can remove this code. I assume it will be fixed eventually.

Reply:
I spend some time trying to come up with a quick solution for this because I do agree that it's always great to know when things become unnecessary in an ignore list but I couldn't come up with a solution I would like as it was a little messy to try to keep track of warning still happening while only performing the check for ruby >= 3.2
If you don't mind, I'll merge this PR and try to address the warning in the gem itself to avoid spending too much time on this. If something goes wrong with the upstream fix I'll open a separate PR with a way to track warning status